collection-scripts/pre-install.sh

```diff
@@ -22,10 +27,22 @@ apply_helper_pod() {
 # Add Ready nodes to the list
 nodes=$(oc get nodes --no-headers | awk '/\yworker\y/{print $1}')
 
-# storing storagecluster name
-storageClusterPresent=$(oc get storagecluster -n "${ns}" -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
-# checking for mcg standalone cluster
-reconcileStrategy=$(oc get storagecluster -n "${ns}" -o go-template='{{range .items}}{{.spec.multiCloudGateway.reconcileStrategy}}{{"\n"}}{{end}}')
+# Check if the storagecluster CRD exists
+# Running the `oc get storagecluster` command directly throws error:
+# `error: the server doesn't have a resource type "storagecluster"`
+CRD_EXISTS=$(oc get crd | grep -c "storageclusters.ocs.openshift.io")
+if [ "$CRD_EXISTS" -eq 0 ]; then
```

Comment:
same as above

Reply:
I have used `-c` which will result in integer output only, so I think this should be fine, please revisit the commands.